### PR TITLE
Conan 0.9.3

### PR DESCRIPTION
**Pull request by lasote**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/conan.rb [hunk 1]
```diff
@@ -11,7 +11,11 @@ class Conan < Formula
     sha256 "aae8d317e2e9add74d893d0183c3f93539f98b81f59f67bed1d1f2a9466198bd" => :mavericks
   end
 
-  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on :python
+  resource "backport_ipaddress" do
```
Comment (UniqMartin) [inline Formula/conan.rb]: Please leave an empty line between the `depends_on` and the `resource` line. Also, why did you drop the `if MacOS.version <= :snow_leopard` on the `depends_on :python` line?

File: Formula/conan.rb [hunk 2]
```diff
@@ -33,16 +42,41 @@ class Conan < Formula
     sha256 "e043c8d32527607223652021ff648fbb394d5e19cba9f1a698670b338c9d782b"
   end
 
+  resource "cryptography" do
+    url "https://pypi.python.org/packages/a9/5b/a383b3a778609fe8177bd51307b5ebeee369b353550675353f46cb99c6f0/cryptography-1.4.tar.gz"
+    sha256 "bb149540ed90c4b2171bf694fe6991d6331bc149ae623c8ff419324f4222d128"
+  end
```
Comment (UniqMartin) [inline Formula/conan.rb]: Does this also mean the formula now depends on `openssl` and thus needs a `depends_on "openssl"` line? At least that's my interpretation of the build failure where it fails to find an OpenSSL header …

File: Formula/conan.rb [hunk 3]
```diff
@@ -75,7 +129,7 @@ class Conan < Formula
 
   def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
-    %w[boto bottle cfgparse colorama fasteners monotonic passlib patch PyJWT PyYAML requests six].each do |r|
+    %w[backport_ipaddress boto bottle cffi cfgparse colorama cryptography enum34 fasteners idna ipaddress monotonic ndg-httpsclient passlib patch pyasn pyasn1 pycparser PyJWT pyOpenSSL PyYAML requests six].each do |r|
```
Comment (UniqMartin) [inline Formula/conan.rb]: If the order of these doesn't matter, let's simplify this by doing something like:

```ruby
resources.each do |r|
  r.stage do
    system "python", *Language::Python.setup_install_args(libexec/"vendor")
  end
end
```

If the order is important, let's at least make this a bit more readable by formatting the `%w[` style array like this:

```ruby
%w[
  backport_ipaddress
  boto
  bottle
  cffi
  (remaining resources)
].each do |r|
  # …
```

**Comment by lasote** [comment]
Hi, thanks for the comments and your time. We realize that 0.9.3 is a bad version, so I'm closing the PR.

**Pull request closed without merging**